healthtools/scrapers/clinical_officers.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
             "name", "reg_date", "reg_no", "valid_dates",
             "address", "qualifications", "id",
         ]
-
+        self.es_index = ES["index"]
```

Comment:
@Gathondu Figured as much but can change it the same way in test coz it's only one and leave out for the rest of the scrapers.